python/ml4ir/applications/ranking/tests/test_base.py

```diff
@@ -71,6 +72,10 @@ def tearDown(self):
         # Delete other temp directories
         file_io.rm_dir(os.path.join(self.root_data_dir, "csv", "tfrecord"))
 
+        # Clear memory
+        tf.keras.backend.clear_session()
```

Comment:
@darshshah Adding this for better clean up. Clears any memory held by keras. But this will not help for tf2.0 sadly. I still think it is a clean practice to have this for when we upgrade tensorflow.